app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Become a secondary teacher 
+subject: secondary
+title_paragraph: |-
+  <p>
+  As a secondary teacher, you'll play a pivotal role in students' lives. You'll see them grow from children to young adults, inspiring them to become passionate about your subject and shaping their futures.</p>
+  
+description: |-
+    Find out how to become a secondary teacher, including what you'll teach and what funding is available to help you train.
```

Comment:
Not sure whether it's officially secondary teacher or secondary school teacher, but I prefer secondary school teacher, as secondary teacher sounds like a second choice almost?